<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Opciones del sensor</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/sigrdap/icons/configsensor3.png</normaloff>:/sigrdap/icons/configsensor3.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
font-size: 10pt;
font-family: &quot;Verdana&quot;;
font-weight: bold;
}

QPushButton:enabled{
	color: #ecf0f1;
	background-color: #3498db;
	padding: 5px;
	border: none;
}

QPushButton:disabled{
	color: #ecf0f1;
	background-color: #bdc3c7;
	padding: 5px;
	border: none;
}

QPushButton:hover{
	padding: 5px;
	color: white;
	background-color: #2980b9;
}


QListWidget{
/*background-color: #5d5d5d;*/
background-color: #3498db;
color: #FFFFFF;
selection-color: #54626e;
selection-background-color: #FFFFFF;
outline: none;
}

QListWidget::item:selected:active{
    background-color: #ecf0f1;
	color: black;
	border: none;
	outline: none;
}

QListWidget::item:selected:!active{
    background-color: #ecf0f1;
	color: black;
	border: none;
	outline: none;
}

QListWidget::item:!selected:hover{
	background-color: #54626e;
	/*color: #5bade4;*/
}

QComboBox, QDateEdit, QAbstractSpinBox{
	color: #2c3e50;
	subcontrol-origin: padding;
	subcontrol-position: top right;
	border: none;
	padding: 5px;
}

QComboBox:down-arrow, QDateEdit:down-arrow, QAbstractSpinBox:down-button{
	image: url(:General/icons/downarrow-a.png);
	width: 20px;
	height: 20px;
}

QAbstractSpinBox:down-button{
		padding-right: 10px;
}

QComboBox:down-arrow:hover, QDateEdit:down-arrow:hover, QAbstractSpinBox:down-button:hover{
	image: url(:General/icons/downarrow.png);
	width: 20px;
	height: 20px;
}

QAbstractSpinBox:up-button{
	image: url(:General/icons/uparrow-a.png);
	width: 20px;
	height: 20px;
	padding-right: 10px;
}

QAbstractSpinBox:up-button:hover{
	image: url(:General/icons/uparrow.png);
	width: 20px;
	height: 20px;
}

QComboBox:drop-down, QDateEdit:drop-down, QAbstractSpinBox:drop-down{
	 subcontrol-origin: padding;
	 subcontrol-position: top right;
	 width: 30px;
	 color: white;
	 border-left-width: 0px;
	 border-left-color: darkgray;
	 border-left-style: solid; /* just a single line */
	 border-top-right-radius: 3px; /* same radius as the QComboBox */
	 border-bottom-right-radius: 3px;
	 padding-right: 5px;
 }

QComboBox:enabled, QDateEdit:enabled, QAbstractSpinBox:enabled{
	background-color: #3498db;
}

QComboBox:disabled, QDateEdit:disabled, QAbstractSpinBox:disabled{
	background-color: #adc1ce;
}

QComboBox:hover, QDateEdit:hover, QAbstractSpinBox:hover{
	background-color: #2980b9;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="kraken">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>65</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #3498db;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="logo">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>:/Varios/icons/logosig2-i.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="botonCerrar">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">:enabled{
	color: black;
	border: none;
	padding: 5px;
}

:disabled{
	color: black;
}

:pressed{
	background-color: black;
}

:hover{
	background-color: #e74c3c;
	border: none;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/General/icons/closeicon-d.png</normaloff>:/General/icons/closeicon-d.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_7">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="listaOpciones">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">:item{
	height: 50px;
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="textElideMode">
        <enum>Qt::ElideNone</enum>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>Alertas</string>
        </property>
        <property name="toolTip">
         <string>Configurar alarmas</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/VentanaConfiguracion/icons/alarma.png</normaloff>
          <selectedoff>:/VentanaConfiguracion/icons/alarma_active.png</selectedoff>
          <selectedon>:/VentanaConfiguracion/icons/alarma_active.png</selectedon>:/VentanaConfiguracion/icons/alarma.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Intervalo</string>
        </property>
        <property name="toolTip">
         <string>Configurar intervalo de env√≠o de datos</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/VentanaConfiguracion/icons/intervalo.png</normaloff>
          <selectedoff>:/VentanaConfiguracion/icons/intervalo_active.png</selectedoff>
          <selectedon>:/VentanaConfiguracion/icons/intervalo_active.png</selectedon>:/VentanaConfiguracion/icons/intervalo.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fecha y hora</string>
        </property>
        <property name="toolTip">
         <string>Configurar fecha y hora</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/VentanaConfiguracion/icons/datetime.png</normaloff>
          <selectedoff>:/VentanaConfiguracion/icons/datetime_active.png</selectedoff>
          <selectedon>:/VentanaConfiguracion/icons/datetime_active.png</selectedon>:/VentanaConfiguracion/icons/datetime.png</iconset>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="styleSheet">
        <string notr="true">QWidget#scrollAreaWidgetContents{
	background-color: #ecf0f1;
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>355</width>
          <height>493</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
